tgui/packages/tgui/interfaces/NtosMain.js

```diff
@@ -58,17 +59,17 @@ export const NtosMain = (props, context) => {
             </Stack>
           </Section>
         )}
-        {!!(cardholder && show_imprint) && (
```

Comment:
Why would you show login if there's no ``show_imprint``? You can't log in?

Reply:
It's not a login, it's information on the card, and it's a button to eject it from the machine, so that there's more than just alt-clicking the computer that's there to take out the card.

Reply:
Isn't that what ``login`` below does?

Reply:
No, login is the registered name in the machine. Here, the use-case is to display the information about the card in the slot, basically.